pkg/bootstrap/stage.go

```diff
@@ -166,7 +170,36 @@ func extractFromDir(dir, prefix string, img v1.Image, imgName string) error {
 	if err := extract(imgName, tempDir, prefix, r); err != nil {
 		return err
 	}
-	return os.Rename(tempDir, dir)
+	if err := os.Rename(tempDir, dir); err != nil && err != os.ErrExist {
+		return err
+	} else if err == nil {
```

Comment:
I don't think this else is necessary.

Reply:
the rest of the code here should work only if there's an os.ErrExist, shouldnt continue when there's no returned errors

Reply:
```suggestion
	} else if err != nil {
```

Reply:
Right, if we renamed it successfully then we're done (can return) instead of continuing on to read/rename the files.